gaia-component.js

```diff
@@ -160,9 +159,7 @@ var base = {
         if (this.lightStyle) { this.appendChild(this.lightStyle); }
       },
 
-      get: function() {
-        return textContent.get();
-      }
+      get: textContent.get
```

Comment:
thanks for reminding that. I was wondering the issue is cause by extra `()`, but in real chrome test it still chokes, so I'd rollback the syntax.